pkg/config/config.go

```diff
@@ -70,6 +70,7 @@ type Root struct {
 type SMTP struct {
 	Addr            string        `required:"true" default:"0.0.0.0:2500" desc:"SMTP server IP4 host:port"`
 	Domain          string        `required:"true" default:"inbucket" desc:"HELO domain"`
+	ProxyHosts      []string      `desc:"Accept network connection from PROXY host(s)"`
```

Comment:
I'm not sure how proxylistener works, would it be more crrect to say "Only accept network connections from PROXY host(s)" ?

Reply:
That is correct, as any other host connecting and using "proxy protocol" could fake connection connecting ip.

Reply:
hm i need to fix that part... it is broken :)

Reply:
```
6:46PM INF Starting SMTP session module=smtp remote=10.10.10.10:2222 session=1
```
from different host ... i need to check who is remote host (this was checked from remote host that should not be able to use proxy protocol...)
and this is by the way way to test if proxy protocol works (log entry)